conda/meta.yaml

```diff
@@ -1,48 +1,42 @@
 {% set data= load_setup_py_data() %}
 package:
-  name: "ts-dream"
+  name: ts-dream
   version: {{ data.get('version') }}
 
 source:
   path: ../
 
 build:
-  noarch: generic
+  noarch: python
   script: {{ PYTHON }} -m pip install --no-deps --ignore-installed .
-  entry_points:
-    - run_dream = lsst.ts.dream.csc:run_dream
 
 test:
   requires:
-    - ts-conda-build
-    - ts-dream-common
+    - ts-conda-build =0.4
     - ts-idl {{ idl_version }}
```

Comment:
ts-idl is deprecated. This is a good time to remove the line. ts-idl was replaced with ts-xml, but dream doesn't use ts-xml either